changelog.txt

```diff
@@ -5,6 +5,7 @@
 * Fix - Resolved an issue with saving plugin settings when bank descriptor value is invalid.
 * Add - Include Stripe API version in logs.
 * Fix - Issue with rendering Sepa on checkout page when card is disabled in non-UPE mode.
+* Tweak - Removed "Early Access" pill and "Disable" button from dropdown menu.
```

Comment:
```suggestion
* Tweak - Removed the "Early Access" pill and "Disable" option from the Stripe payment methods dropdown menu.
```

Just a minor suggestion but I was thinking that adding context to where this disable button is would help avoid confusion to anyone reading the changelog.